src/messenger-shortcuts.js

```diff
@@ -78,6 +92,16 @@ function last(arr) {
   return arr.length === 0 ? undefined : arr[arr.length - 1];
 }
 
+function getCode(index) {
+  return bind[index].charCodeAt(0);
+}
+
+function dismiss() {
+	if(menuOpened) {
```

Comment:
The dismiss function just toggles the boolean menuOpened, which just makes it so if the help dialog is already opened, you can't open it again. And sorry about the spacing!!! I'll fix it.